docs/manual/docs/administrator-guide/managing-users-and-groups/creating-group.md

```diff
@@ -30,10 +30,49 @@ Additional settings:
 - **Editing**: Specifies which groups can edit the metadata record.
 - **Notify**: Determines which groups are notified when a file managed by GeoNetwork is downloaded.
 
+## Group Type
+
+Administrators can define the type of group being created. There are three group types available, each with specific characteristics and use cases:
+
+### 1. Workspace Group
+
+- **Description**: This is the default group type. It allows the group to own metadata records and have privileges assigned to them.
+- **Use Case**: Suitable for groups that need full control over metadata records, including creation, import, and transfer.
+
+### 2. Record Privilege Group
+
+- **Description**: This group type can have privileges assigned to specific metadata records but cannot own metadata records.
+- **Use Case**: Ideal for groups that require access to metadata records without ownership rights.
+
+### 3. System Privilege Group
+
+- **Description**: This group type is used to manage system-level privileges. It cannot own metadata records or have privileges assigned to them.
+- **Use Case**: Designed for administrative purposes where system-level access is required without metadata ownership.
```

Comment:
I think it is relevant to explain why only can assign this type of groups to `Registered user` profile.